operations/helm/charts/mimir-distributed/templates/validate.yaml

```diff
@@ -130,3 +130,8 @@
     {{- fail "memcached-results.enabled has been renamed to results-cache.enabled" }}
   {{- end }}
 {{- end }}
+
+
+{{- if eq ($calculatedConfig.alertmanager).external_url "/alertmanager" }}
+  {{- fail "Do not set alertmanager.external_url to the old invalid value /alertmanager in either mimir.config or mimir.structuredConfig. The value must start with proper http:// or https:// scheme." }}
+{{- end }}
```

Comment:
I'm not too sure about validating config here that mimir already validates. Aren't we just duplicating logic like that?